### DISCUSSION TITLE
Is SqlPower real?

### DISCUSSION THREAD

**Discussion by sj14** [AllenDang/giu#594] [General]
I was looking for a GUI library for a SQL DB Browser. I found giu and a nice pictures https://github.com/AllenDang/giu/blob/f9430b5a874c7c691ba245e04d4b5123d96c0dc4/screenshots/SqlPower.png

Is this tool real?

**Comment by gucio321** [comment]
why not. There is no code for that, but you can easily write it!

**Reply by sj14** [reply]
Thanks for the answer. I was curious if there was already this product shown in the screenshot, but from your answer, I understand that it was only some GUI showcase demo.

**Reply by AllenDang** [reply]
@sj14 No, it's not just a show demo but a tool be used for years in my company. It just not open sourced. So far I can tell is giu is fully capable to create a db browser.
